docker-compose.yml

```diff
@@ -68,6 +68,215 @@ services:
     links:
       - s3
 
+  pg_delete_before_name_find_full_test:
```

Comment:
Try to find out: is it possible to create targets with parameters in .yml files?